packages/cli/src/lib/autorun/autorun-command.unit.test.ts

```diff
@@ -16,31 +17,34 @@ vi.mock('@code-pushup/core', async () => {
   };
 });
 
+const cli = (options = {}) =>
+  yargsCli(
+    objectToCliArgs({
+      _: 'autorun',
+      verbose: true,
+      config: '/test/code-pushup.config.ts',
+      'persist.outputDir': '/test',
+      ...options,
+    }),
+    {
+      ...DEFAULT_CLI_CONFIGURATION,
+      commands: [yargsAutorunCommandObject()],
+    },
+  );
+
 describe('autorun-command', () => {
   beforeEach(() => {
     vol.fromJSON(
       {
-        'my-report.json': JSON.stringify(MINIMAL_REPORT_MOCK),
+        'report.json': JSON.stringify(MINIMAL_REPORT_MOCK),
```

Comment:
```suggestion
        'my-report.json': JSON.stringify(MINIMAL_REPORT_MOCK),
```
This change was here on purpose to check that the report file name is changed from the default if provided by the CLI.